scripts/watchdog

```diff
@@ -84,7 +86,9 @@ function enable_watchdog()
         fi
     elif isMLNX; then
         debug "Calling MLNX WD enable"
-        # call watchdog api for mlnx
+        if [[ -x /usr/bin/hw-management-wd.sh  ]]; then
```

Comment:
@stepanblyschak what if the condition is False? Should we handle this case? Any warning/error?